sla_per_use_system_test.py

```diff
@@ -0,0 +1,394 @@
+        test_name = 'sla_per_user_basic'
+        stress_duration = 10 # minutes
+        shares = [200, 1000]
+        read_users = []
+        error = None
+        for share in shares:
+            read_users.append({'user': User(session=session, name='user%d' % share, password='user%d' % share),
+                               'role': Role(session=session, name='role%d' % share),
+                               'service_level': ServiceLevel(session=session, name='sla%d' % share, service_shares=share)})
+
+        expected_shares_ratio = self.calculate_shares_ratio(read_users[1]['service_level'].service_shares,
+                                                            read_users[0]['service_level'].service_shares)
+
+        self.log.debug('============== Start {} test'.format(test_name))
+        self.create_auths(entities_list_of_dict=read_users)
+
+        read_cmds = [self.STRESS_READ_CMD.format(n=self.n, user=read_users[0]['user'].name,
+                                                 password=read_users[0]['user'].name,
+                                                 pop='seq=1..%d' % (self.n),
+                                                 duration='%dm' % stress_duration, threads=250),
+                     self.STRESS_READ_CMD.format(n=self.n, user=read_users[1]['user'].name,
+                                                 password=read_users[1]['user'].name,
+                                                 pop='seq=1..%d' % (self.n),
+                                                 duration='%dm' % stress_duration, threads=250)
+                     ]
+
+        try:
+            start_time = time.time()
+
+            results = self.run_stress_get_results(read_cmds=read_cmds,
+                                                  users=read_users,
+                                                  waited_cpu_min=self.WAITED_CPU_MIN)
+            end_time = time.time()
+
+            self.validate_scheduler_runtime(start_time=start_time, end_time=end_time,
+                                            read_users=read_users, expected_ratio=expected_shares_ratio)
+
+            self.assertTrue(results, msg='Not received cassandra-stress results')
+
+            self.log.debug('Validate cassandra-stress ops deviation')
+            actual_shares_ratio = self.calculate_shares_ratio(results[read_users[1]['user'].name],
+                                                              results[read_users[0]['user'].name])
+            self.validate_deviation(expected_ratio=expected_shares_ratio,
+                                    actual_ratio=actual_shares_ratio, msg='Validate cassandra-stress ops.')
+
+            self.log.debug('============== Finish {} test'.format(test_name))
+        except Exception as e:
+            self.log.error(e)
+            error = e.message
+        finally:
+            self.clean_auth(entities_list_of_dict=read_users)
+            return error
+
+    def throughput_latency_load(self, session):
```

Comment:
add test_ to the name.
change name to something more clear, maybe what we used in the presentation.

Reply:
Also we need the ability to easy duplicate and test with different values.

Reply:
Changed